Lesson9/anna/index.html

```diff
@@ -6,6 +6,395 @@
     <title>This is a placeholder HTML for assignment 9</title>
```

Comment:
Styles file is not added to the HTML right now, don't forget to add the link element:
`<link rel="stylesheet" href="./styles.css">`